src/reports/components/report-sections/no-failed-instances-congrats.tsx

```diff
@@ -3,27 +3,13 @@
 import * as React from 'react';
 
 import { NamedFC } from 'common/react/named-fc';
-import { InlineImage, InlineImageType } from '../inline-image';
-import {
-    reportCongrats,
-    reportCongratsHead,
-    reportCongratsInfo,
-    reportCongratsMessage,
-    reportCongratsScreen,
-} from './no-failed-instances-congrats.scss';
+import { reportCongratsHead, reportCongratsInfo, reportCongratsMessage } from './no-failed-instances-congrats.scss';
 
 export const NoFailedInstancesCongrats = NamedFC('NoFailedInstancesCongrats', () => {
     return (
-        <div className={reportCongrats} key="report-congrats">
-            <div>
-                <InlineImage imageType={InlineImageType.AdaLaptop} alt="" />
```

Comment:
did you check the fast pass html report? I think remove this `InlineImage` here will also make the image to not show on the report... is that intended?